src/main/java/seedu/student/model/student/Faculty.java

```diff
@@ -57,6 +57,23 @@ public static boolean isValidFaculty(String test) {
         return Arrays.asList(facultyArray);
     }
 
+
+    /**
+     * Returns true if the given string is a faculty in the enum.
+     *
+     * @param givenFaculty the string to be checked.
+     * @return true if givenFaculty is a valid faculty, false otherwise.
+     */
+
+    public static Boolean isExist(String givenFaculty) {
+        for (FacultyAbbreviation facultyAbbreviation : FacultyAbbreviation.values()) {
+            if (givenFaculty.equals(facultyAbbreviation.name())) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
What's the difference between this method and `isValidFaculty` method?

Reply:
Good point! It turns that I can use 'isValidFaculty' method instead of coding a new method.